decoder_test.go

```diff
@@ -153,6 +153,26 @@ func TestDecoder_Uint32(t *testing.T) {
 	assert.Equal(t, 0, d.remaining())
 }
 
+func TestDecoder_Int32(t *testing.T) {
+
+	buf := new(bytes.Buffer)
+	enc := NewEncoder(buf)
+	enc.writeInt32(int32(342))
```

Comment:
In fact, can you add a negative test like in [Line #116](https://github.com/eoscanada/eos-go/pull/66/files#diff-d477aef9bf305a8d89d0973a5b775b8aR116) so we cover both positive and negative numbers.

Once it's changed, I'm going to merge your work.